pip install does not build TS

See title.

After setting up cookie-cutter and running `pip install -e ".[test, examples]" .`, I could not find `node_module` folder and running `npm test` failed because it could not find `karma`.

I manually ran `npm install` and now `npm test works`.

The error message given for `npm test` before running `npm install` is as follows:

```
$ npm test

> ipyneu3D@0.1.0 test /path/to/ipyneu3D
> npm run test:firefox


> ipyneu3D@0.1.0 test:firefox /path/to/ipyneu3D
> karma start --browsers=Firefox tests/karma.conf.js

sh: 1: karma: not found
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! ipyneu3D@0.1.0 test:firefox: `karma start --browsers=Firefox tests/karma.conf.js`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the ipyneu3D@0.1.0 test:firefox script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/tingkai/.npm/_logs/2019-03-14T13_30_48_558Z-debug.log
npm ERR! Test failed.  See above for more details.

```

--- a/README.md
+++ b/README.md
@@ -41,7 +41,7 @@ you should run the tests:
 
 ```bash
 # First install the python package. This will also build the JS packages.
-pip install -e ".[test, examples]" .
+pip install -e ".[test, examples]"
 
 # Run the python tests. This should not give you a few sucessful example tests
 py.test

